jetstream/core/orchestrator.py

```diff
@@ -384,6 +384,10 @@ def __init__(
             self.detokenize_threads,
         )
     )
+    if self.metrics_collector:
```

Comment:
Are you collecting the model load time? Why adding the metric here?

Reply:
Synced with @liurupeng, for our purposes we want to measure the time between server start and when the server is live (when self.live = True).